### PR TITLE
Hide important info from debug mode

### PR DESCRIPTION
**Pull request by diashalabi**

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -58,15 +58,11 @@
     */
 
     'debug_blacklist' => [
-        '_ENV' => [
-            'APP_KEY',
-            'DB_PASSWORD',
-        ],
+        '_COOKIE' => array_keys($_COOKIE),
```
Comment (fatihalp) [inline config/app.php:61]: didnt understand logic?

Comment (diashalabi) [reply config/app.php:61]: When you add an element to this array it replaces it's value with ************.

screenshot-alibaba loc-2021 12 06-10_36_58

Comment (fatihalp) [reply config/app.php:61]: i see but why array_keys($_COOKIE)

Comment (fatihalp) [reply config/app.php:61]: it's my cookies then no problem acutally :)

**Review by spektra2147 — APPROVED** [review]

**Review by muammertop — APPROVED** [review]

**Pull request merged**